src/base/bittorrent/session.cpp

```diff
@@ -269,6 +288,8 @@ Session::Session(QObject *parent)
     , m_maxUploadsPerTorrent(BITTORRENT_SESSION_KEY("MaxUploadsPerTorrent"), -1, lowerLimited(0, -1))
     , m_isUTPEnabled(BITTORRENT_SESSION_KEY("uTPEnabled"), true)
     , m_isUTPRateLimited(BITTORRENT_SESSION_KEY("uTPRateLimited"), true)
+    , m_utpMixedMode(BITTORRENT_SESSION_KEY("uTPMixedMode"), m_isUTPEnabled, clampValue(0, 1))
```

Comment:
>Actually, the link points to:
...

@LordNyriox, this code can be useful in case we have decided store enums in config file as strings. But, IIRC, @sledgehammer999 and @Chocobo1 prefer to use numbers so such enum wrapping is unnecessary.